types/styled-components/v3/index.d.ts

```diff
@@ -90,6 +90,7 @@ export type StyledInterface = ThemedStyledInterface<DefaultTheme>;
 export interface DefaultTheme {}
 
 export interface ThemeProviderProps<T> {
+    children: React.ReactElement;
```

Comment:
@eps1lon why React.ReactElement here instead of ReactNode?

Reply:
https://github.com/styled-components/styled-components/blob/v3.0.2/src/models/ThemeProvider.js#L22

Reply:
Though it's nullable so let's get this in 
```suggestion
    children?: React.ReactElement | undefined | null;
```